lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml

```diff
@@ -29,6 +29,10 @@ perform_action_item:
     button_text: <item_text>
   action: click_actions_button
   action: click_button_text
+perform_silence:
```

Comment:
Does `submit_changes` meet your requirement?

Reply:
perform_silence is fine, create_silence is fine too, the function name should be easy to understand and show what is the function do

Reply:
no more comments from me

Reply:
In 4.4 and previous release, the button named create. Frome 4.5, the button named Silence

Reply:
changed code, action perform_silence will call action submit_changes
thx